crates/sui-graphql-rpc/src/types/epoch.rs

```diff
@@ -12,52 +12,131 @@ use super::transaction_block::{TransactionBlock, TransactionBlockFilter};
 use super::validator_set::ValidatorSet;
 use async_graphql::connection::Connection;
 use async_graphql::*;
+use sui_indexer::models_v2::epoch::StoredEpochInfo;
 
-#[derive(Clone, Debug, PartialEq, Eq, SimpleObject)]
-#[graphql(complex)]
+#[derive(Clone, Debug)]
 pub(crate) struct Epoch {
-    /// The epoch's id as a sequence number that starts at 0 and it is incremented by one at every epoch change
-    pub epoch_id: u64,
+    pub stored_epoch_info: StoredEpochInfo,
+}
+
+#[Object]
+impl Epoch {
     /// The epoch's protocol version
     #[graphql(skip)]
```

Comment:
Instead of marking this as `#[graphql(skip)]` in the `#[Object]` impl block (which TIL you can even do that) can you put it in its own impl block below this one so it's clear what is part of the GraphQL interface and what is not?